http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientCallingPatternIntegrationTest.java

```diff
@@ -197,8 +213,14 @@ public void checkAllCombinations(@FromDataPoints("EventLoop") int eventLoopSize,
             CrtResource.waitForNoResources();
 
             float numSeconds = (float) ((System.currentTimeMillis() - start) / 1000.0);
+            float numSeconds_test = (float) ((System.currentTimeMillis() - start_test) / 1000.0);
```

Comment:
It's confusing with two different timing statements, the numSeconds and numSeconds_test. Do we need both? If not, which makes most sense? If we do can we refactor to make it clearer?

Reply:
this was added for testing integ test trigerred by this PR , now its no longer required